scripts/deploy:push

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env bash
 set -e
-set -x
```

Comment:
Ah, yeah, I'll add the rest. 

It's too spammy in the common case, especially when you're echoing things (double echo)